### ISSUE TITLE
Deep links doesn't seem to work

### ISSUE THREAD

**Issue by alexander-potemkin** [vector-im/element-meta#1946]
#### What would you like to do?

Provide an option to open a client (desktop, mobile) with prefilled server and, ideally, only a login option (without registration).
I can make all that kind of customizations on web client easily, but for desktop & mobile I need clone, reproduce build, change things, which is all quite resource intense task.

#### Why would you like to do it?

To provide an option to easily log in to the server for non-tech savvy users.
Something of the kind seems to be at [Matrix.org server](https://matrix.to/#/#element-web:matrix.org), but it doesn't work for a customer server.

#### How would you like to achieve it?

I tried:
- element://vector/webapp/#/room/%23discuss%3Amyserver.com
- element://vector/webapp/#/user/@nickname:myserver.com

None of them works.

Desktop doesn't pick it up on a fresh start (only if it was already started), but when I click login/register, it's back to matrix.org server.
iOS app just ignores that altogether.

So, having one working way to call the client with predefined server and, ideally, option to login (not register), via something like  element://vector/webapp/#/room/%23discuss%3Amyserver.com?register=0 would be ideal.

### Have you considered any alternatives?

Yes. I've tried to make a custom build (offered to [provide it back to Element](https://github.com/vector-im/element-meta/discussions/1806), but with no luck).

### Additional context

That feels an important thing for self-hosted servers.

**Comment by t3chguy** [comment]
> with prefilled server

https://github.com/vector-im/element-web/issues/5469

**Comment by alexander-potemkin** [comment]
I believe it's rather https://github.com/vector-im/element-web/pull/17824, but anyway - they are both for the web, which can be nicely fine-tuned via config file, as opposed to iOS & Android, which makes me belive it's more important on the mobile platforms.

**Comment by t3chguy** [comment]
@alexander-potemkin that is a pull request which fixes the issue I linked...

**Comment by alexander-potemkin** [comment]
@t3chguy , thank you. Is it correct, that the fix is for web app only, and it won't affect iOS & Android, Desktop?

**Comment by t3chguy** [comment]
@alexander-potemkin it will affect Desktop too. Element Desktop = Element Web + Electron + Seshat + Keytar

**Comment by alexander-potemkin** [comment]
@t3chguy thanks, I understand.
My main concern are mobiles - Android & iOS specifically. As they are locked by app store owners and making custom build is troublesome, doesn’t seem to be of any interest from Element mobile team, unfortunately.

**Comment by t3chguy** [comment]
I suggest making issues on the mobile repos, they don't tend to keep an eye on element-meta as far as I can see

**Comment by alexander-potemkin** [comment]
Yeah, it seems that way... My [other suggestion](https://github.com/vector-im/element-meta/discussions/1806) didn't even get any attention :(

Thanks, opened a separate issue!
